### DISCUSSION TITLE
GUI for generic or himbaechel

### DISCUSSION THREAD

**Discussion by KelvinChung2000** [YosysHQ/nextpnr#1393] [Q&A] [answered]
I would like to use GUI for either himbaechel or generic arch. What is required for the architecture to generate something similar to the ice40 demo?

**Comment by gatecat** [comment, marked as answer]
See https://github.com/YosysHQ/nextpnr/pull/1295 - @mmicko do you think you'll have a moment sometime to look at my very belated review of that PR?

**Reply by mmicko** [reply]
@gatecat will adress it tomorrow, if you have some more comments please do add. Thanks
